### ISSUE TITLE
Duplicate imports, when the name is used multiple times 

### ISSUE THREAD

**Issue by kbauer** [lyz-code/autoimport#129]
## Description

When an unimported symbol occurs multiple times, duplicate import statements may be produced.

## Steps to reproduce

1. Create a file `demo.py`
    ```
    def f(x):
        return numpy.sin(x) + numpy.cos(x) + numpy.tan(x)
    ```
2. Execute `autoimport demo.py`.

## Current behavior

The file will contain a duplicate import of `numpy`.

```
import numpy
import numpy
import numpy

def f(x):
    return numpy.sin(x) + numpy.cos(x) + numpy.tan(x)
```

## Desired behavior

The module should be imported only once.

```
import numpy

def f(x):
    return numpy.sin(x) + numpy.cos(x) + numpy.tan(x)
```

## Environment

```
>>> python -c "import autoimport.version; print(autoimport.version.version_info())"
           autoimport version: 0.7.0
               python version: 3.9.5 (default, May 18 2021, 19:34:48)  [GCC 7.3.0]
                     platform: Linux-4.12.14-lp151.28.91-default-x86_64-with-glibc2.26
```

**Comment by lyz-code** [comment]
Hi @kbauer, thanks for taking the time for opening the issue.

I've fixed it in `0.7.1`, please check if you can no longer reproduce the bug.

As a side note, remember that it's recommended to use [isort and black](https://lyz-code.github.io/autoimport/#usage) after autoimport to produce a cleaner output.

**Comment by kbauer** [comment]
Can confirm. I don't see the issue anymore (tested with `0.7.2`).
